src/plugins/usage_collection/server/collector/collector_set.ts

```diff
@@ -89,7 +102,7 @@ export class CollectorSet {
     return [...this.collectors.values()].find((c) => c.type === type);
   };
 
-  public isUsageCollector = (x: UsageCollector | any): x is UsageCollector => {
+  public isUsageCollector = (x: AnyUsageCollector | any): x is AnyUsageCollector => {
```

Comment:
An aside: What benefit do we get from using the `AnyUsageCollector` type rather than the class `UsageCollector`?

Reply:
Avoiding `UsageCollector<any, any>` everywhere 😅